pkg/costmodel/allocation_helpers.go

```diff
@@ -21,7 +21,7 @@ import (
 const CPU_SANITY_LIMIT = 512
 
 // Sanity Limit for PV usage, set to 10 PB, in bytes for now
-const PV_USAGE_SANITY_LIMIT_BYTES = 10737418240
```

Comment:
`10737418240 = 10 * (1024 ^ 3)` which is 10 GiB

Reply:
(And if you try to reach petabytes with an int it overflows.)

Reply:
@nikovacevic I like the fixes for clarity here, thanks! So you're saying the following wouldn't overflow? Why not?

```
const PV_USAGE_SANITY_LIMIT_BYTES = 10.0 * PiB
```

Reply:
Correct. That doesn't overflow because it's a different data type, `float64`. Max `float64` is `1.7E+308` haha. Max `int` on the other hand is `2147483647`, which is only ~2GiB. (In fact... it's _precisely_ 1 less than 2GiB, which is `2147483648`! Why could that be? This is your math homework :wink: )

Reply:
Inferred because of `10.0` versus `10`, and the type of `PiB` is also `float64` for that same reason. Here's a fun "gotcha" example to play around with: https://go.dev/play/p/4nomozb9-t6

Reply:
And you pass your math assignment! Haha. (It's somewhat more complicated than this, but the gist is correct: a 32-bit integer can only support 2^32 unique numbers, or 4294967296 numbers -- which ends up being -2147483648, ..., 0, ... 2147483647 when encoded with [two's complement](https://en.wikipedia.org/wiki/Two%27s_complement).)